docs/custom-flows/manage-sms-based-mfa.mdx

```diff
@@ -0,0 +1,374 @@
+  'use client';
+
+  import * as React from 'react';
+  import { useUser } from '@clerk/nextjs';
+  import { BackupCodeResource, PhoneNumberResource } from '@clerk/types';
+  import Link from 'next/link';
+
+  // Display phone numbers reserved for MFA
+  const ManageMfaPhoneNumbers = () => {
+    const { user } = useUser();
+
+    if (!user) return null;
+
+    // Check if any phone numbers are reserved for MFA
+    const mfaPhones = user.phoneNumbers
+      .filter((ph) => ph.verification.status === 'verified')
+      .filter((ph) => ph.reservedForSecondFactor)
+      .sort((ph: PhoneNumberResource) => (ph.defaultSecondFactor ? -1 : 1));
+
+    return (
+      <>
+        <h2>Phone numbers reserved for MFA</h2>
+        {mfaPhones.length === 0 ? (
+          <p>No phone numbers reserved for MFA</p>
+        ) : (
+          <ul>
+            {mfaPhones.map((phone) => {
+              return (
+                <li key={phone.id} style={{ display: 'flex', gap: '10px' }}>
+                  <p>
+                    {phone.phoneNumber} {phone.defaultSecondFactor && '(Default)'}
+                  </p>
+                  <div>
+                    <button
+                      onClick={() =>
+                        phone.setReservedForSecondFactor({ reserved: false })
+                      }
+                    >
+                      Disable for MFA
+                    </button>
+                  </div>
+
+                  {!phone.defaultSecondFactor && (
+                    <div>
+                      <button onClick={() => phone.makeDefaultSecondFactor()}>
+                        Make default
+                      </button>
+                    </div>
+                  )}
+
+                  <div>
+                    <button onClick={() => phone.destroy()}>
+                      Remove from account
+                    </button>
+                  </div>
+                </li>
+              );
+            })}
+          </ul>
+        )}
+      </>
+    );
+  };
+
+  // Display phone numbers that are not reserved for MFA
+  const ManageAvailablePhoneNumbers = () => {
+    const { user } = useUser();
+
+    if (!user) return null;
+
+    // Check if any phone numbers aren't reserved for MFA
+    const availalableForMfaPhones = user.phoneNumbers
+      .filter((ph) => ph.verification.status === 'verified')
+      .filter((ph) => !ph.reservedForSecondFactor);
+
+    // Reserve a phone number for MFA
+    const reservePhoneForMfa = async (phone: PhoneNumberResource) => {
+      // Set the phone number as reserved for MFA
+      await phone.setReservedForSecondFactor({ reserved: true });
+      // Refresh the user information to reflect changes
+      await user.reload();
+    };
+
+    return (
+      <>
+        <h2>Phone numbers that are not reserved for MFA</h2>
+        {availalableForMfaPhones.length === 0 ? (
+          <p>
+            None - <Link href="/account/add-phone">Add a new phone number</Link>
```

Comment:
A button to add a number should always be present. Once a user adds one number, this button should no longer show. Let's move this back to the bottom.